README.md

```diff
@@ -1,3 +1,16 @@
+# Web version of Vange-rs
+
+Compile Web (Debug):
+
+```sh
+PATH=$PATH:/home/caiiiycuk/rust/rust/build/x86_64-unknown-linux-gnu/stage0/lib/rustlib/x86_64-unknown-linux-gnu/bin/ LD_LIBRARY_PATH=$LD_LIBRARY_PATH:/home/caiiiycuk/rust/rust/build/x86_64-unknown-linux-gnu/stage0/lib/ cargo build  --target wasm32-unknown-unknown --bin road --verbose && wasm-bindgen --out-dir html5/ --target web --keep-debug target/wasm32-unknown-unknown/debug/road.wasm
```

Comment:
I guess it would be better to change `/home/caiiiycuk` to `$HOME` ?

Reply:
Yeah, I created this pr to show changes I did. So in final version I will fix it. Btw thanks for looking